idaes/gas_solid_contactors/unit_models/tests/test_BFB.py

```diff
@@ -217,11 +220,269 @@ def test_build(self, iron_oc):
     def test_dof(self, iron_oc):
         assert degrees_of_freedom(iron_oc) == 0
 
+    @pytest.fixture(scope="class")
+    def iron_oc_unscaled(self, iron_oc):
+        import copy
+        m = copy.deepcopy(iron_oc)
```

Comment:
As I commented in the previous PR, I would suggest you just rebuild the model rather than copying.